common/dynamicconfig/constants.go

```diff
@@ -2510,6 +2518,12 @@ const (
 	// Default value: 30 minutes
 	ESAnalyzerBufferWaitTime
 
+	// IsolationGroupCfgUpdateFrequency
+	// KeyName: system.isolationGroupCfgUpdateFrequency
+	// Value type: Duration
+	// Default value: 2 minutes
+	IsolationGroupCfgUpdateFrequency
```

Comment:
How about naming it `system.isolationGroupStateRefreshInterval`?

Reply:
That's a better name